views/components/ConfigurationCard/index.jsx

```diff
@@ -73,7 +74,7 @@ const ConfigurationCard = ({
           {ownsConfig ? (
             <>
               <IconButton
-                onClick={() => openNewWindow(routes.editConfiguration(_id))}
+                onClick={() => navigate(routes.editConfiguration(_id))}
```

Comment:
I think it'll be better to pass in a prop here like `onEditConfig` just like we have the other handler props. Then this component doesn't need to know about the imperative steps and this could change to `onClick={() => onEditConfig(config)}`

Reply:
[5680b34](https://github.com/AMP-SCZ/dpdash/pull/317/commits/5680b345846866194d0bca9196b33cc650150d38)